pennylane/templates/embeddings/amplitude.py

```diff
@@ -133,9 +133,6 @@ def __init__(self, features, wires, pad_with=None, normalize=False, do_queue=Tru
     def num_params(self):
         return 1
 
-    def adjoint(self):
```

Comment:
Standard decomposition pathway would get us to the same point. Doesn't need to be explicitly specified.

`Adjoint(AmplitudeEmbedding)` => `Adjoint(QubitStateVector)` => `Adjoint(MottonStatePreparation)`.